scrapy/utils/display.py

```diff
@@ -3,24 +3,55 @@
 """
 
 import sys
+import platform
 from pprint import pformat as pformat_
 
 
 def _colorize(text, colorize=True):
     if not colorize or not sys.stdout.isatty():
         return text
     try:
+        import pygments.util
+        from pygments.formatters import get_formatter_by_name
         from pygments import highlight
-        from pygments.formatters import TerminalFormatter
         from pygments.lexers import PythonLexer
-        return highlight(text, PythonLexer(), TerminalFormatter())
     except ImportError:
         return text
 
+    if sys.platform == "win32" and platform.release() == "10":
+        if platform.version() >= "10.0.14393":
```

Comment:
- Shouldn’t this be a single `if` statement?
- Is `platform.release() == "10"` needed, given `platform.version() >= "10.0.14393"`?
- Isn’t using `>=` for string comparison problematic here?
- I don’t think it is obvious from the code why `10.0.14393` is hardcoded there. Maybe we need a comment explaining it?

Reply:
> * Shouldn’t this be a single `if` statement?
> 
> * Is `platform.release() == "10"` needed, given `platform.version() >= "10.0.14393"`?
yes, it should be single `if` statement and there is no need for `platform.release( ) == "10"`

>     * Isn’t using `>=` for string comparison problematic here? 
It is problematic. So, now I have used `packaging.version.parse()` for version comparison.